Lesson_Materials/Lecture_04_Handling_Errors/index.html

```diff
@@ -92,28 +92,25 @@
 					</div>
 					<div class="hbox" >
 						<code class="code-60pc"><pre>
-class add;
+class kernel;
```

Comment:
It looks like you've changed the name here, but the usage is still saying `parallel_for<add>`. I find the name add clearer.